### ISSUE TITLE
Documentation build/deploy is failing

### ISSUE THREAD

**Issue by phutchins** [dcgtc/dgrants#388]
The build in Fleek is failing which is preventing the deploy from happening.

Here is the build output:
```
Deploy started at 12:08:00 PM 10/13/2021

12:08:01 PM 10/13/2021: Deploy started
12:08:03 PM 10/13/2021: Docker image: 'phutchins/volta-base:0.0.2'
12:08:03 PM 10/13/2021: Build command: 'yarn && yarn build'
12:08:03 PM 10/13/2021: Triggering build execution...
12:09:25 PM 10/13/2021: yarn install v1.22.5
12:09:25 PM 10/13/2021: [1/4] Resolving packages...
12:09:25 PM 10/13/2021: [2/4] Fetching packages...
12:09:44 PM 10/13/2021: info fsevents@2.3.2: The platform "linux" is incompatible with this module.
12:09:44 PM 10/13/2021: info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
12:09:44 PM 10/13/2021: info fsevents@1.2.13: The platform "linux" is incompatible with this module.
12:09:44 PM 10/13/2021: info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
12:09:44 PM 10/13/2021: [3/4] Linking dependencies...
12:09:44 PM 10/13/2021: warning " > file-loader@6.2.0" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
12:09:44 PM 10/13/2021: warning " > url-loader@4.1.1" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
12:09:44 PM 10/13/2021: warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react@3.0.0-alpha.40" has unmet peer dependency "@types/react@>= 16.8.0 < 18.0.0".
12:09:44 PM 10/13/2021: warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react > @algolia/autocomplete-preset-algolia@1.2.2" has unmet peer dependency "@algolia/client-search@^4.9.1".
12:09:54 PM 10/13/2021: [4/4] Building fresh packages...
12:09:55 PM 10/13/2021: Done in 29.94s.
12:09:55 PM 10/13/2021: yarn run v1.22.5
12:09:55 PM 10/13/2021: $ docusaurus build
12:09:57 PM 10/13/2021: [en] Creating an optimized production build...
12:09:58 PM 10/13/2021: ℹ Compiling Client
12:09:58 PM 10/13/2021: ℹ Compiling Server
12:10:21 PM 10/13/2021: ✔ Client: Compiled successfully in 22.73s
12:10:23 PM 10/13/2021: ✔ Server: Compiled successfully in 25.21s
12:10:23 PM 10/13/2021: {
12:10:23 PM 10/13/2021: file: 'assets/css/styles.58014dd2.css',
12:10:23 PM 10/13/2021: message: "Css Minimizer Plugin: Unexpected '}' at assets/css/styles.58014dd2.css:25:28236. ",
12:10:23 PM 10/13/2021: compilerPath: 'client'
12:10:23 PM 10/13/2021: }
12:10:23 PM 10/13/2021: {
12:10:23 PM 10/13/2021: file: 'assets/css/styles.58014dd2.css',
12:10:23 PM 10/13/2021: message: "Css Minimizer Plugin: Invalid property name '&--dark{--ifm-hero-text-color' at assets/css/styles.58014dd2.css:25:28191. Ignoring. ",
12:10:23 PM 10/13/2021: compilerPath: 'client'
12:10:23 PM 10/13/2021: }
12:10:23 PM 10/13/2021: {
12:10:23 PM 10/13/2021: file: 'assets/css/styles.58014dd2.css',
12:10:23 PM 10/13/2021: message: "Css Minimizer Plugin: Invalid selector '}#nprogress' at assets/css/styles.58014dd2.css:25:28236. Ignoring. ",
12:10:23 PM 10/13/2021: compilerPath: 'client'
12:10:23 PM 10/13/2021: }
12:10:23 PM 10/13/2021: {
12:10:23 PM 10/13/2021: file: 'assets/css/styles.58014dd2.css',
12:10:23 PM 10/13/2021: message: "Css Minimizer Plugin: Invalid selector '}#nprogress' at assets/css/styles.58014dd2.css:25:28236. Ignoring. ",
12:10:23 PM 10/13/2021: compilerPath: 'client'
12:10:23 PM 10/13/2021: }
12:10:35 PM 10/13/2021: Unable to build website for locale "en".
12:10:35 PM 10/13/2021: Error: Docusaurus found broken links!
12:10:35 PM 10/13/2021: Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
12:10:35 PM 10/13/2021: Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
12:10:35 PM 10/13/2021: It looks like some of the broken links we found appear in many pages of your site.
12:10:35 PM 10/13/2021: Maybe those broken links appear on all pages through your site layout?
12:10:35 PM 10/13/2021: We recommend that you check your theme configuration for such links (particularly, theme navbar and footer).
12:10:35 PM 10/13/2021: Frequent broken links are linking to:
12:10:35 PM 10/13/2021: - /docs/intro
12:10:35 PM 10/13/2021: Exhaustive list of all broken links found:
12:10:35 PM 10/13/2021: - On source page path = /404.html:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/archive:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/first-blog-post:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/long-blog-post:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/mdx-blog-post:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/tags:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/tags/docusaurus:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/tags/facebook:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/tags/hello:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/tags/hola:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /blog/welcome:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/tags:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /markdown-page:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/contributors-guide/add-to-cart-checkout:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/contributors-guide/contributors-faq:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/funders-guide/contributors-faq:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/funders-guide/fund-grant-round:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/grant-owners-guide/claim-payouts:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/grant-owners-guide/create-edit-grants:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/grant-owners-guide/grant-owners-faq:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/how-does-dgrants-work/architecture-overview:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/how-does-dgrants-work/concepts:
12:10:35 PM 10/13/2021: -> linking to (/docs/contributors-guide/add-to-cart-checkout) (resolved as: /docs/how-does-dgrants-work/(/docs/contributors-guide/add-to-cart-checkout))
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/how-does-dgrants-work/smart-contracts:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/introduction/intro:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/introduction/path-to-decentralization:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/matching-pool-funder-guide/add-matching-pool:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: - On source page path = /docs/matching-pool-funder-guide/matching-pool-funders-faq:
12:10:35 PM 10/13/2021: -> linking to /docs/intro
12:10:35 PM 10/13/2021: at Object.reportMessage (/workspace/doc/docs-web/node_modules/@docusaurus/utils/lib/index.js:308:19)
12:10:35 PM 10/13/2021: at Object.handleBrokenLinks (/workspace/doc/docs-web/node_modules/@docusaurus/core/lib/server/brokenLinks.js:142:17)
12:10:35 PM 10/13/2021: at async buildLocale (/workspace/doc/docs-web/node_modules/@docusaurus/core/lib/commands/build.js:160:5)
12:10:35 PM 10/13/2021: at async tryToBuildLocale (/workspace/doc/docs-web/node_modules/@docusaurus/core/lib/commands/build.js:32:20)
12:10:35 PM 10/13/2021: at async Object.mapAsyncSequencial (/workspace/doc/docs-web/node_modules/@docusaurus/utils/lib/index.js:263:24)
12:10:35 PM 10/13/2021: at async build (/workspace/doc/docs-web/node_modules/@docusaurus/core/lib/commands/build.js:68:25)
12:10:35 PM 10/13/2021: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
12:10:35 PM 10/13/2021: error Command failed with exit code 1.
12:10:59 PM 10/13/2021: Error occured during the build.
```

**Comment by mk1r** [comment]
I believe this one is closed by #475 can you confirm @phutchins?

**Comment by jjwoz** [comment]
@phutchins I second what @mk1r said. I believe this is resolved
